substrate/client/executor/common/src/runtime_blob/runtime_blob.rs

```diff
@@ -227,7 +228,7 @@ impl RuntimeBlob {
 	pub fn as_polkavm_blob(&self) -> Option<&polkavm::ProgramBlob> {
 		match self.0 {
 			BlobKind::WebAssembly(..) => None,
-			BlobKind::PolkaVM(ref blob) => Some(blob),
+			BlobKind::PolkaVM((ref blob, _)) => Some(blob),
```

Comment:
```suggestion
			BlobKind::PolkaVM((ref program_blob, _)) => Some(program_blob),
```

Let's make it clear, at least at the variable name level, for now.